frame/support/procedural/src/construct_runtime/expand/event.rs

```diff
@@ -79,7 +79,7 @@ pub fn expand_outer_event(
 			#scrate::RuntimeDebug,
 		)]
 		#[allow(non_camel_case_types)]
-		pub enum Event {
+		pub enum RuntimeEvent {
```

Comment:
Before I go on and start renaming Event into RuntimeEvent, it would be helpful if someone could check if I am doing the right thing. The main change is inside this file, the other changes are just the renaming of the Event. Should I just continue renaming Event into RuntimeEvent, or is there a smarter way to do this?

Reply:
So far so good, but you should also rename some `Event`s to `PalletEvent` depending on where they are used, e.g. I would look closer at the `event_variants` binding and check to see if they require renaming `Event`s to `PalletEvent`s.